doc/development_process/api_lifecycle.rst

```diff
@@ -77,16 +77,70 @@ An API can be declared ``stable`` after fulfilling the following requirements:
 - Stable APIs can get backward compatible updates, bug fixes and security fixes
   at any time.
 
+Introducing incompatible changes
```

Comment:
Shall we somehow present it in the [api lifecycle](https://github.com/zephyrproject-rtos/zephyr/blame/master/doc/development_process/api_lifecycle.rst#L13)?

Reply:
I don't think so, because the figure is for the API as a whole, and not for selected incompatible changes.